GitUI/CommandsDialogs/FormRebase.cs

```diff
@@ -321,18 +330,24 @@ private void OkClick(object sender, EventArgs e)
 
                 Skipped.Clear();
 
+                bool? updateRefChoice = null;
```

Comment:
Why make it tri-state?

Reply:
Config. If config has set value then mixed stars is I haven't overridden value since unchecked is a choice for no-updare-refs while checked is update-refs. So mixed is take config value. I can see the need for it to be tri-state to not be there and instead just read config and set default value from config value.